src/test/resources/passay-test.properties

```diff
@@ -29,3 +29,4 @@ SOURCE_VIOLATION=Password cannot be the same as your %1$s password.
 TOO_LONG=Password must be no more than %2$s characters in length.
 TOO_SHORT=Password must be %1$s or more characters in length.
 TOO_MANY_OCCURRENCES=Password contains %2$s occurrences of the character '%1$s', but at most %3$s are allowed.
+MATCH_BEHAVIOR_STARTS_WITH=begins with
```

Comment:
I think there's an assumption here that may break under some languages: that simply translating one word or phrase like "contains" is sufficient. While that probably produces comprehensible phrases in all languages, I'm not sure it would always produce the most _natural_ phrasing. The root problem is that the code defines english language descriptions, and this use case demonstrates that that is problematic.

Reply:
Provided a similar implementation in the latest commit where uses can choose to define custom messages for any rule that uses `MatchBehavior`.  The format uses the (dot) character as the delimiter to follow the existing convention: `ILLEGAL_CHAR.CONTAINS=my custom message`.

Reply:
Looks much better!